CHANGELOG.md

```diff
@@ -85,6 +85,15 @@ Adding a new version? You'll need three changes:
   [#4749](https://github.com/Kong/kubernetes-ingress-controller/pull/4749)
 - Removed Knative support.
   [#4748](https://github.com/Kong/kubernetes-ingress-controller/pull/4748)
+- Removed support for the `debug-log-reduce-redundancy` CLI flag.
+  [#4688](https://github.com/Kong/kubernetes-ingress-controller/pull/4688)
+- The "text" logging format has changed. "json" should be used for
+  machine-parseable logs.
+  [#4688](https://github.com/Kong/kubernetes-ingress-controller/pull/4688)
```

Comment:
I can see that JSON log format changes the timestamp format, from 

    "time":"2023-09-21T23:15:15Z"

to

    "time":"2023-09-22T00:28:13.006Z"

Which is a subtle change but might break some log ingestion systems/rules/etc.

This should definitely be noted in here as for some this will be a breaking change.

Reply:
Good catch. Switched to RFC3339, which does ISO8601 with second precision. Console format because that's what the tests run in, but the time encoder is the same:

```
2023-10-10T00:49:13Z	info	setup	Initializing Dataplane Client	{"v": 0}
2023-10-10T00:49:13Z	info	setup	Starting Admission Server	{"v": 0}
```